spec/rubocop/cop/style/word_array_spec.rb

```diff
@@ -82,10 +82,6 @@
       expect_no_offenses('%w(one two three)')
     end
 
-    it 'does not register an offense for array with one element' do
-      expect_no_offenses('["three"]')
-    end
-
```

Comment:
It feels a weird to include that in the `word_array` and `symbol_array` cops, since that's really a test of the `array_syntax.rb` mixin. There's also already a test that sort of coverts the non-zero `MinSize`, though it may be better to just be more explicit about it.

Thoughts?

```ruby
  it 'detects right value for MinSize to use for --auto-gen-config' do
      inspect_source(<<-RUBY.strip_indent)
        [:one, :two, :three]
        %i(a b c d)
      RUBY

      expect(cop.offenses.size).to eq(1)
      expect(cop.messages).to eq(['Use `%i` or `%I` for an array of symbols.'])
      expect(cop.config_to_allow_offenses).to eq('EnforcedStyle' => 'percent',
                                                 'MinSize' => 4)
    end
```

Reply:
I added two specs to explicitly test the MinSize > 1 behavior